databases/models/ChannelMessages.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class ChannelMessages extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  ChannelMessages.init(
+    {
+        id: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true
+        },
+        message_id: {
+            type: DataTypes.UUID,
+            allowNull: false
+        },
+        channel_id:{
+            type: DataTypes.UUID,
+            allowNull: false,
+        },
+        user_id: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        message: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        is_read_by_admin: {
+            type: DataTypes.BOOLEAN,
+            defaultValue: false
+        },
+        createdAt: {
+            type: DataTypes.DATE,
+            field: "created_at",
+        },
+        updatedAt: {
+            type: DataTypes.DATE,
+            field: "updated_at",
+        },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've correctly used the `field` option to map the `createdAt` and `updatedAt` fields to snake_case equivalents in the database. This helps maintain consistency between the JavaScript code and the underlying database schema.



<!-- This is an auto-generated comment by OSS CodeRabbit -->